docs/develop/understanding-the-javascript-api-for-office.md

```diff
@@ -45,11 +44,16 @@ But your start-up code must not call any Office.js APIs until the library is ful
 - [Initialize with Office.onReady()](#initialize-with-officeonready)
 - [Initialize with Office.initialize](#initialize-with-officeinitialize)
 
-For information about the differences in these techniques, see [Major differences between Office.initialize and Office.onReady()](#major-differences-between-officeinitialize-and-officeonready). For more details about the sequence of events when an add-in is initialized, see [Loading the DOM and runtime environment](loading-the-dom-and-runtime-environment.md).
+> [!TIP]
+> We recommend that you use `Office.onReady()` instead of `Office.initialize`. Although `Office.initialize` is still supported, using `Office.onReady()` provides more flexibility.
```

Comment:
Consider adding a clarification to "provides more flexibility". Like, "provides more flexibility with X". It'll answer the biggest question without forcing people to click the following link.